berkeley-function-call-leaderboard/SUPPORTED_MODELS.md

```diff
@@ -44,7 +44,7 @@ Below is a comprehensive table of models supported for running leaderboard evalu
 |nova-lite-v1.0| Function Calling|
 |nova-macro-v1.0| Function Calling|
 |command-r-plus-FC | Function Calling|
-|command-r-plus | Prompt|
```

Comment:
The diff may be a little confusing here -  we've removed the **prompt** version of command-r-plus, kept the FC version, and added the FC version of `command-r7b-12-2024`